segmenters/nlp/Sentencizer/__init__.py

```diff
@@ -8,8 +8,19 @@
 class Sentencizer(BaseSegmenter):
     """
     :class:`Sentencizer` split the text on the doc-level into sentences on the chunk-level with a rule-base strategy.
-        The text is split by the punctuation characters listed in ``punct_chars``.
-        The sentences that are shorter than the ``min_sent_len`` or longer than the ``max_sent_len`` after stripping will be discarded.
+    The text is split by the punctuation characters listed in ``punct_chars``.
+    The sentences that are shorter than the ``min_sent_len`` or longer than the ``max_sent_len`` after stripping will be discarded.
+
+    :param min_sent_len: the minimal number of characters,
+        (including white spaces) of the sentence, by default 1.
+    :param max_sent_len: the maximal number of characters,
+        (including white spaces) of the sentence, by default 1e5.
+    :param punct_chars: the punctuation characters to split on.
```

Comment:
I think for `punct_chars` may be good to show an example, (like ',.') is `,`  OR `.`

Reply:
But we have so many, should we add just a few on the docstring?

Reply:
what I mean is that the argument I do not find it so intuitive to say that by giving a string with different punctuaction chars ',.:' means that any of the characters in the string will be used to split a document

Reply:
What you think of this: '

`the punctuation characters to split on, for example, but not limited to '.', ',', '!'`